ui/components/multichain/asset-picker-amount/asset-picker-modal/AssetList.tsx

```diff
@@ -32,20 +37,28 @@ type AssetListProps = {
   isTokenDisabled?: (
     token: AssetWithDisplayData<ERC20Asset> | AssetWithDisplayData<NativeAsset>,
   ) => boolean;
+  network?: NetworkConfiguration;
+  isTokenListLoading?: boolean;
 };
 
 export default function AssetList({
   handleAssetChange,
   asset,
   tokenList,
   isTokenDisabled,
+  network,
```

Comment:
When would we not require a network here?

Reply:
When the cross-chain swaps experience is initially loaded a network won't be set for the destination (Bridge to) asset picker: https://www.figma.com/design/IuOIRmU3wI0IdJIfol0ESu/Cross-Chain-Swaps?node-id=159-28072&m=dev